rest/route/host_create.go

```diff
@@ -259,7 +259,7 @@ func (h *containerLogsHandler) Run(ctx context.Context) gimlet.Responder {
 	} else {
 		options.ShowStdout = true
 	}
-	logs, err := h.sc.GetDockerLogs(ctx, h.host.ExternalIdentifier, parent, settings, options)
+	logs, err := h.sc.GetDockerLogs(ctx, h.host.Id, parent, settings, options)
```

Comment:
I'm surprised by this. Isn't external identifier the DNS name, whereas id is typically "sir-" or "i-"?

Reply:
Apparently since we create the container by assigning host.Id as the name, we can now reference it by the name we assigned, which feels cleaner.